### PR TITLE
Add transactions to wallet page

### PR DESCRIPTION
**Pull request by Yulatun**
### Description

Please provide a brief description of the changes you have made in your PR.

### Type of change

- [x] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [x] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)
- [x] Chakra components/constants/hooks have been used wherever possible

> If not explain here

- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change

Add screenshots from the [current deployed state](https://sandbox-money-app.herokuapp.com/) of the changed components here.
image

#### Screenshots/Video after change

Add screenshots with the changes applied here.
image
image

update (after adding mt):  
image

### CODE REVIEW

File: src/components/ExpenseItem/ExpenseItem.jsx [hunk 1]
```diff
@@ -10,30 +10,15 @@ import {
 
 import { NoteIcon } from '@/assets';
 
-export const ExpenseItem = ({ transaction, onEdit, onDelete }) => {
+export const ExpenseItem = ({ transaction, onEdit, onDelete, version }) => {
```
Comment (kalashnikau) [inline src/components/ExpenseItem/ExpenseItem.jsx:13]: do it boolean? like `isShortView` or `isLongDisplayVersion` or something like that

File: src/components/ExpenseItem/ExpenseItem.jsx [hunk 2]
```diff
@@ -64,6 +49,31 @@ export const ExpenseItem = ({ transaction, onEdit, onDelete }) => {
             <IconButton onClick={onDelete} icon={<DeleteIcon />}></IconButton>
```
Comment (kalashnikau) [inline src/components/ExpenseItem/ExpenseItem.jsx:49]:
```suggestion
            <IconButton onClick={onDelete} icon={<DeleteIcon />} />
```

Comment (Yulatun) [reply src/components/ExpenseItem/ExpenseItem.jsx:49]: done

File: src/components/ExpenseItem/ExpenseItem.jsx (hunk 1 above)
Comment (Yulatun) [reply src/components/ExpenseItem/ExpenseItem.jsx:13]: done

**Review by kalashnikau — APPROVED** [review]

**Review by HarryPotter-4000 — DISMISSED** [review]

**Review by kalashnikau — APPROVED** [review]

File: src/components/ExpenseItem/ExpenseItem.jsx [hunk 3]
```diff
@@ -58,12 +43,37 @@ export const ExpenseItem = ({ transaction, onEdit, onDelete }) => {
...
+
+  return (
+    <>
+      <HStack
+        justify="space-around"
+        w="100%"
+        boxShadow="lg"
+        bg={bgMain}
+        color={textColor}
+        paddingY={5}
```
[the remark this answers is not in the document]
Comment (Yulatun) [reply src/components/ExpenseItem/ExpenseItem.jsx:64]: done

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -25,7 +25,7 @@ export const Header = (isOpen) => {
       <Flex
         align="center"
         justify="space-between"
-        maxWidth="1200px"
+        maxW="container.xl"
```
Comment (lazzzaroni) [inline src/components/Header/Header.jsx:28]: This prop set max-width of the header to '1280px', to keep our max-width consistent

**Review by lazzzaroni — APPROVED** [review]

**Review by kalashnikau — APPROVED** [review]

**Pull request merged**